CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing to @ftrack/api
+
+Thank you for wanting to contribute to our Javascript API!
+
+Please clone this repository, and follow these instructions to make sure you are in a clean spot before coding:
+
+Using node version as reflected in `.nvmrc`, run:
+
+- yarn install
+- yarn test
+
+If the tests passes, you should be good to go. Please submit a PR with your changes and we will look at them as soon as possible.
+
+## Publishing a release
+
+_This is only relevant for ftrack employees when preparing and publishing a new release._
+
+To publish a new release, follow these steps:
+
+Locally:
+
+- `git switch -c release/<new-release-tag>`
```

Comment:
What purpose does the release branch serve here?

Reply:
When running `npm version <new-release-tag>`, a new commit is created. Since we restrict push access to main branch, we cannot push it directly so we need to create a PR with the new commit. By creating release branches we standardize how to do this.

We might be able to also create a workflow that does all this for us if we want to.

Reply:
Ah, I see. Previously we allowed commits on the main branch for some people but trusted them to only push releases and documentation changes. Sounds like an good solution for now.